packages/user/src/model/invitations/handlers/createInvitation.ts

```diff
@@ -88,7 +85,15 @@ const createInvitation = async (
 
     try {
       invitation = await service.create(invitationCreateInput);
-    } catch {
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    } catch (error: any) {
```

Comment:
can't we just catch error and send it as an response? why do we need 2 error message one is error.message and check your input?